src/sentry/sentry_metrics/indexer/postgres_v2.py

```diff
@@ -165,7 +34,7 @@ def _get_db_records(self, db_keys: KeyCollection) -> Any:
 
         return StringIndexerTable.objects.filter(query_statement)
 
-    def bulk_record(self, org_strings: Mapping[int, Set[str]]) -> Mapping[int, Mapping[str, int]]:
+    def bulk_record(self, org_strings: Mapping[int, Set[str]]) -> KeyResults:
```

Comment:
Logic of this method is mostly unchanged, but since I migrated away from `merge_results` operating on mappings and am now operating on `KeyResults` objects, some intermediate values were able to go away

Reply:
Do you mind updating the doc string for this method to reflect your changes?

Reply:
will do, good catch